src/main/scala/algolia/objects/Query.scala

```diff
@@ -220,7 +222,9 @@ case class Query(
       "restrictIndices" -> restrictIndices.map(_.mkString(",")),
       "restrictSources" -> restrictSources.map(_.mkString(",")),
       /* Browse */
-      "cursor" -> cursor
+      "cursor" -> cursor,
+      /* Virtual Indices */
+      "relevancyStrictness" -> relevancyStrictness.map(_.toString)
```

Comment:
I think relevancyStrictness is a number, so has to be this, no? Is there a test?

```suggestion
      "relevancyStrictness" -> relevancyStrictness
```

Reply:
@Haroenv this is related to how the serialization of the query is done, which by the end will be serialized to a map of string key / string value:
`def toQueryParam: Map[String, String] `
(you will notice all fields are mapped to string at some point)

Reply:
the other parameters that are .map( seem to be arrays though, do numbers have .map?

Reply:
Yes, `hitsPerPage: Option[Int]` for example is mapped using `"hitsPerPage" -> hitsPerPage.map(_.toString)`